### PR TITLE
UI Polish

### PR DESCRIPTION
**Pull request by fabian202**

### CODE REVIEW

File: src/components/molecules/InputSearch/index.js [hunk 1]
```diff
@@ -34,6 +34,9 @@ const InputStyled = styled(Input)`
   border-left: 0;
   border-top: 0;
   border-right: 0;
+  padding: 0;
+  height: 2rem;
+  margin-top: 0rem;
```
Comment (normandanielmq) [inline src/components/molecules/InputSearch/index.js:39]: zero values do not require the unit.

File: src/components/molecules/MemberReports/index.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import Select, { components } from 'react-select';
 import styled, { css } from 'styled-components';
 import { FaChevronDown, FaStar, FaRegStar, FaInfoCircle } from 'react-icons/fa';
+import { A } from 'hookrouter';
```
Comment (normandanielmq) [inline src/components/molecules/MemberReports/index.js:5]: Is this used in some place?

File: src/components/pages/ProgramListPage/index.js [hunk 3]
```diff
@@ -326,12 +331,18 @@ const StyledPagination = styled(Pages)`
...
+
+  > div {
+    padding: 0;
+
+    &:last-child {
     text-align: right;
     button {
       width: auto;
     }
   }
```
Comment (normandanielmq) [inline src/components/pages/ProgramListPage/index.js:344]: Missing indentation

File: src/components/molecules/InputSearch/index.js (hunk 1 above)
Comment (fabian202) [reply src/components/molecules/InputSearch/index.js:39]: In this case need to override it

File: src/components/molecules/MemberReports/index.js (hunk 2 above)
Comment (fabian202) [reply src/components/molecules/MemberReports/index.js:5]: Yeah!

`const Link = styled(A)`
  color: ${({ theme }) => theme.palette.grayscale[1]};
`;`

File: src/components/molecules/InputSearch/index.js (hunk 1 above)
Comment (normandanielmq) [reply src/components/molecules/InputSearch/index.js:39]: I meant adding just `margin-top: 0;` is enough.

**Pull request merged**